### PR TITLE
Add search box for outbox and PI

### PR DESCRIPTION
**Pull request by GretaD**
Continuing https://github.com/nextcloud/mail/pull/6546/
Fixes https://github.com/nextcloud/mail/issues/7474

The first version of search box on mail is only shown on Inbox, but it should be shown in all mailboxes. So, i removed it from Mailbox.vue component and added it on Mailboxthread

### CODE REVIEW

**Comment by GretaD** [comment]
cc @sazanof its still a draft but just so you are aware :)

**Comment by ChristophWurst** [comment]
> Fixes #7461

Wrong ticket?

**Review by sazanof — CHANGES_REQUESTED** [review]
@GretaD super cool!

How do you think, may be also edit styles in SearchMessages.vue in this PR too?

```
.search-messages {
	min-height: 52px;
	margin: 3px 0 0 52px;
	border-right: 1px solid var(--color-border);
	position: relative;
	display:flex;
	align-items: center;
```
image

**Review by sazanof — APPROVED** [review]
Tested and works! Nice =)

___________________

As for the "Outbox" filter, it is difficult to implement it from what is. Outgoing messages use a local table in the database, with a different structure. 
So we need to do a separate search for them.

In addition, under normal circumstances, this folder should be empty.

**Comment by GretaD** [comment]
> In addition, under normal circumstances, this folder should be empty.

exactly, or with a little number of emails that can be easily found, thus no need for a search field, at least for now.

**Review by st3iny — APPROVED** [review]
Tested and works.

File: src/components/Outbox.vue [hunk 1]
```diff
@@ -45,6 +43,7 @@
 					:key="message.id"
 					:message="message" />
 			</AppContentList>
+		</template>
 		</div>
```
[the remark this answers is not in the document]
Comment (st3iny) [reply src/components/Outbox.vue:47]: To fix an ESLint issue.

**Pull request merged**